TRIAGING.md

```diff
@@ -0,0 +1,69 @@
+2. Announcements: If there are any announcements to be made they will happen at the start of the meeting.
+3. Review of New Issues: The meetings always start with reviewing all untriaged [issues](https://github.com/issues?q=is%3Aissue+label%3Auntriaged+repo%3Aopensearch-project%2Fopensearch-build+repo%3Aopensearch-project%2Fhelm-charts+repo%3Aopensearch-project%2Fansible-playbook+repo%3Aopensearch-project%2Fopensearch-ci+repo%3Aopensearch-project%2Fopensearch-cluster-cdk+repo%3Aopensearch-project%2Fterraform-provider-opensearch+repo%3Aopensearch-project%2Fopensearch-devops+repo%3Aopensearch-project%2Fopensearch-build-libraries+is%3Aopen+) for below repositories:
+    1. [opensearch-build](https://github.com/opensearch-project/opensearch-build)
+    1. [opensearch-ci](https://github.com/opensearch-project/opensearch-ci)
+    1. [opensearch-build-libraries](https://github.com/opensearch-project/opensearch-build-libraries)
+    1. [helm-charts](https://github.com/opensearch-project/helm-charts)
+    1. [terraform-provider-opensearch](https://github.com/opensearch-project/terraform-provider-opensearch)
+    1. [ansible-playbook](https://github.com/opensearch-project/ansible-playbook)
+    1. [opensearch-cluster-cdk](https://github.com/opensearch-project/opensearch-cluster-cdk)
+    1. [opensearch-devops](https://github.com/opensearch-project/opensearch-devops)
+4. Member Requests: Opportunity for any meeting member to ask for consideration of an issue or pull request.
+5. Untriaged Items: Review any [issues](https://github.com/issues?q=is%3Aissue+label%3Auntriaged+repo%3Aopensearch-project%2Fopensearch-build+repo%3Aopensearch-project%2Fhelm-charts+repo%3Aopensearch-project%2Fansible-playbook+repo%3Aopensearch-project%2Fopensearch-ci+repo%3Aopensearch-project%2Fopensearch-cluster-cdk+repo%3Aopensearch-project%2Fterraform-provider-opensearch+repo%3Aopensearch-project%2Fopensearch-devops+repo%3Aopensearch-project%2Fopensearch-build-libraries+is%3Aopen+) that might have had the 'untriaged' label removed but require additional triage discussion.
+6. Open Discussion: Allow for members of the meeting to surface any topics without issues filed or pull request created.
+
+
+There is no specific ordering within each category.
+
+If you have an issue you would like to discuss but do not have the ability to attend the entire meeting please attend when is best for you and signal that you have an issue to discuss when you arrive.
+
+### Do I need to have already contributed to the project to attend a triage meeting?
+
+No, all are welcome and encouraged to attend. Attending the Backlog & Triage meetings is a great way for a new contributor to learn about the project as well as explore different avenues of contribution.
+
+### What if I have an issue that is almost a duplicate, should I open a new one to be triaged?
```

Comment:
```suggestion
### What if my issue is almost a duplicate? Should I open a new one?
```

Reply:
Keeping it elaborate for now. We can revisit later.